src/fsharp/service/ServiceAssemblyContent.fs

```diff
@@ -241,7 +262,7 @@ module AssemblyContentProvider =
                           processIdents (fullCompiledIdents |> String.concat ".") fullCompiledIdents)
                      |> Option.toList ])
 
-    let rec private traverseEntity contentType (parent: Parent) (entity: FSharpEntity) = 
```

Comment:
Why remove `private`? Just curious.

Reply:
It's already private because the signature file doesn't expose it anyway.